pkgs/by-name/fr/frog-protocols/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  fetchFromGitHub,
+  lib,
+  meson,
+  ninja,
+  nix-update-script,
```

Comment:
```suggestion
```
I believe using this right now can make the package auto-update to 0.0.1. We should be using `unstableGitUpdater` instead

Reply:
But I don't think we want to stay on an unstable version forever. I only chose to do so for now, as there were some minor fixes already.

Reply:
We could always revert this on the next stable release (or omit it now). I'm just concerned about triggering a downgrade in a nixpkgs-update PR

Minor nitpick though honestly

Reply:
If r-ryantm opens a bad PR, we can just convert it into a draft PR to keep people from merging it. AFAIK that's what's usually done in cases where the update script did something undesirable.

Reply:
Fair enough!

Reply:
FWIW I agree with @getchoo that `unstableGitUpdater` should be used right now (or the `updateScript` should be left blank), but I don't consider that blocking.